plugins/Denizen/scripts/utility/integerize.dsc

```diff
@@ -0,0 +1,31 @@
+integerize:
+    type: procedure
+    debug: false
+    description: Multiplies a decimal or integer by 10 until it is an integer.
```

Comment:
I know this description looks dumb, but this is literally what it does. If you input an integer it returns the inputted integer.